src/main/java/com/nike/cerberus/service/MetadataService.java

```diff
@@ -151,6 +152,19 @@ public String getSdbId(SDBMetadata sdbMetadata) {
         return id;
     }
 
+    /**
+     * Overrides the owner of the SDB. This is an admin function.
+     * @param name the name of the SDB
+     */
+    public void overrideOwner(String name, String newOwner, String adminUser) {
```

Comment:
Why is this in the metadata service?

Seems like a method that should in the safe deposit box service exclusively.

Reply:
updated